### ISSUE TITLE
CSS style missing responsive design styles

### ISSUE THREAD

**Issue by shunguoy** [IBMa/equal-access#1277]
One of the responsive design features is to use CSS @media orientation to apply different styles to different devices and orientations. The current Checker engine CSS styles are only resolved to the default web browser orientation (landscape), and missing the other orientation (portrait, or mobile default).  The CSS styles can be applied to multiple @media features, for example, orientation (landscape vs. portrait), screen width/height (e.g., >600px, < 400px), and screen resolution (eg., 150dpi), or any combinations.  The engine needs to consider at least web and mobile default orientation.

The following is a test case:

```
<html lang="en">
	<head>
		<title>Page with some content</title>
		<style>
                      @media screen, print {
				div {
					  background-color:  purple;
				}
			}
                      @media (width > 600px) {
                                div {
                                           font-size: 24;
                                 }
                       }

                      @media (400px <= width <= 600px) {
                              body {
                                       font-size: 18;
                               }
                      }
		    @media ( orientation:  portrait ) {
				div {
					  background-color:  red;
				}
		     }
                     @media ( orientation:  landscape ) {
				div {
					  background-color:  blue;
				}
		      }

		</style>
	</head>
	<body>
		<div>
			   Page Content to be displayed
                 </div>  
         </body>
</html>
```

**Comment by shunguoy** [comment]
this case can be handled in the js event as page orientation or screen size changes. lose for now.
